frontend/src/components/ScheduledEventsRunHistoryTableCard/ScheduledEventsRunHistoryTableCard.tsx

```diff
@@ -84,12 +84,11 @@ export const ScheduledEventsRunHistoryTableCard: FunctionComponent<ComponentProp
             title={`Filter By Status = ${status ? statusMapping[status] : 'All'}`}
             variant="danger"
             size="sm"
+            onSelect={onFilter}
           >
-            <Dropdown.Item onSelect={onFilter} eventKey="">
-              All
-            </Dropdown.Item>
+            <Dropdown.Item eventKey="">All</Dropdown.Item>
             {Object.keys(statusMapping).map((status: RunInstanceStatus) => (
-              <Dropdown.Item key={status} onSelect={onFilter} eventKey={status}>
+              <Dropdown.Item key={status} eventKey={status}>
```

Comment:
This is also happening on develop, so I don't think it has just been introduced, but looking into it..

Reply:
@edwinmp I have failed to get rid of this warning. At the moment the only major difference I see is having react18, because react-bootstrap is still the same version, and I haven't found any documentation on how react18 could affect the DropdownButton. I ran the same on my local develop which is react17 and I still get the same warning

Reply:
This isn't enough to stall this feature ... approving.